### PR TITLE
feat: add EcoscoreData.missingDataWarning field support

### PR DESCRIPTION
**Pull request by FredJul**
I have added the support for the `missing_data_warning` field, which allows the user to know the ecoscore may be improved.

However, I have decided to use a non-nullable boolean field, while the returned json use a `1` or empty value. I'm not sure why the server does not use boolean values but I believe it should be simplified here.

### CODE REVIEW

**Review by MohamedFBoussaid — APPROVED** [review]
All good, Thanks @FredJul

**Comment by FredJul** [comment]
Thanks @MohamedFBoussaid :)

Just wonder if I should wait for another approve before merging it? What is the current process?
Also, do you know when a new version of the lib will be published on pub?

**Comment by MohamedFBoussaid** [comment]
It is always good to have a second review. And regarding the lib release there is no roadmap for the releasing date. 

But I guess @stephanegigandet can help on this. Can you please review the PR ? And maybe we can release a new version with the latest changes?

**Review by peterwvj — APPROVED** [review]
Looks good, thanks!

**Pull request merged**
